Engine/Animation/AnimController.cpp

```diff
@@ -155,6 +155,14 @@ bool AnimController::IsOnState(const std::string& state)
 	return active_state.get()->name_hash == state_hash;
 }
 
+void AnimController::SetSpeed(float speed) const
+{
+	if(active_state)
+	{
+		active_state->speed = speed;
```

Comment:
I understand this is for runtime, so in this case you should be setting the speed for the current playing animation, as this change will affect to all the models with the same state machine. playing_clips[ClipType::ACTIVE].speed is what you want

Reply:
Done!